src/Message/Content/Image.php

```diff
@@ -14,6 +16,17 @@ public function __construct(
     ) {
     }
 
+    public static function fromFile(string $filePath): self
```

Comment:
I would rather check in the ctor and do the logic if the string does not start with http or data and then do the logic. Another idea is to check if it starts with a slash